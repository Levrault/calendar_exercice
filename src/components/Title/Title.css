.title {
  color: $palette-title;
  font-size: 1.1rem;
  font-weight: 300;
  margin-bottom: $space-s;
}
