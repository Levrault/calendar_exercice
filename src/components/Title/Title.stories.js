import React from 'react';
import { storiesOf } from '@storybook/react';
import { withMarkdownNotes } from '@storybook/addon-notes';
import Wrapper from '../../tools/storybook/Wrapper';
import Title from './Title';

storiesOf('Title', module)
  .addDecorator(Wrapper)
  .add('default', withMarkdownNotes(`
    # Title

    Simple month title
    `)(() => (
    <Title>
      This is a title
    </Title>
  )));

