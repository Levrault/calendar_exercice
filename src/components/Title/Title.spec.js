import React from 'react';
import renderer from 'react-test-renderer';
import Title from './Title';

describe('Title component', () => {
  it('should render without props', () => {
    const component = renderer.create(
      <Title>Title</Title>,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
