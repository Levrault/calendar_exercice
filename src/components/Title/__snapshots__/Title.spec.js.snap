// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component should render without props 1`] = `
<h1>
  Title
</h1>
`;
